urwid/main_loop.py

```diff
@@ -1470,8 +1470,7 @@ def _exception_handler(self, loop, context):
             loop.stop()
             if not isinstance(exc, ExitMainLoop):
                 # Store the exc_info so we can re-raise after the loop stops
-                import sys
-                self._exc_info = sys.exc_info()
+                self._exc_info = (type(exc), exc, None)
```

Comment:
Wouldn't it be better to include the traceback with `(type(exc), exc, exc.__traceback__)`? Like mentioned here: https://github.com/urwid/urwid/issues/235#issuecomment-458483802

Reply:
@mback2k I agree, the traceback object should be included.  When I opened this, I was only trying to solve the immediate issue and was following the existing structure (if memory serves, this was a year ago).

Since this has been merged, do you want to make the change or should I open another PR?

Reply:
Please go ahead if you would like to. :-)

Reply:
@nocarryr no need for a new PR.  I'll make the change.